### DISCUSSION TITLE
Implement HTTP status and control of the accessory

### DISCUSSION THREAD

**Discussion by mgx0** [RavenSystem/esp-homekit-devices#2328] [New Features]
Hi there

thanks for an amazing project. I have a question: is there an, or do you plan to implement simple HTTP interface to get status and eventually control the accessories?

i could not find anything. The http server is not running during normal operation and I haven't found a way to turn anything similar in the configs

mqtt client would be also very nice

**Comment by RavenSystem** [comment]
https://github.com/RavenSystem/esp-homekit-devices/wiki/general-configuration#enable-unsecure-homekit-rest-api

**Reply by mgx0** [reply]
Oh crap, how could i overlook this? Thank you very much
